test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Date.Tests.ps1

```diff
@@ -97,6 +97,90 @@ Describe "Get-Date DRT Unit Tests" -Tags "CI" {
         Get-Date -Date $date -UFormat %V | Should -BeExactly $week
     }
 
+    # Using the same test cases as V for ISO week date component parity
+    It "using -uformat 'G' produces the correct output" -TestCases @(
+        @{date="1998-01-02"; year = "1998"},
```

Comment:
I'd suggest adding some more esoteric test cases here. For example, `0325-01-01` and `0055-12-31`